src/cmd/src/metasrv.rs

```diff
@@ -128,6 +130,10 @@ impl TryFrom<StartCommand> for MetaSrvOptions {
             opts.use_memory_store = true;
         }
 
+        if let Some(metrics_addr) = cmd.metrics_addr {
```

Comment:
Could we reuse the http-addr arguments? We only need to add a `/metrics` path to the HTTP api.